external/n2k/CMakeLists.txt

```diff
@@ -82,6 +82,9 @@ target_link_libraries(n2k gputils)
 target_include_directories(n2k PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
 target_include_directories(n2k PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}")
 
+# Install the shared library so installed kotekan can find libn2k at runtime.
```

Comment:
Do we also need Kendrick's gpuutils?

Reply:
This seems to be sufficient - I get no warnings or errors when running an installed kotekan with gpu configs.